plugins/cordova/popup-handler.js

```diff
@@ -107,29 +107,28 @@ PopupHandler.prototype.startHandler = function (event, cb) {
 
   var callbackUrl = urljoin('https:', this.webAuth.baseOptions.domain, '/mobile');
 
-  if (event.url && !(event.url.indexOf(callbackUrl + '#') === 0)) { return; }
+  if (event.url && !(event.url.indexOf(callbackUrl + '#') === 0)) {
+    return;
+  }
 
   var parts = event.url.split('#');
 
   if (parts.length === 1) {
     return;
   }
 
-  var opts = { hash: parts.pop(), _idTokenVerification: false };
+  var opts = { hash: parts.pop() };
```

Comment:
So, we skipping this in Cordova, you know why?

Reply:
I don't we should probably test that with the beta.